src/Http/BaseApplication.php

```diff
@@ -249,22 +249,22 @@ public function getContainer(): ContainerInterface
         if ($this->container) {
             return $this->container;
         }
-        $container = $this->register(new Container());
+        $container = $this->services(new Container());
         foreach ($this->plugins->with('register') as $plugin) {
```

Comment:
I guess this should then be `services` too, with the `BasePlugin::$registerEnabled` property being renamed to `servicesEnabled` accordingly.

Reply:
thats true